docs/documentation/development/bindings/xml-reference.md

```diff
@@ -101,6 +101,16 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>criteria.name</td><td>The name of the context related filter.</td></tr>  
 </table>
 
+Context is used to render different kind of input widgets on front-end. Following contexts are currently supported in PaperUI.
+
+<table><tr><th>Name</th><th>Locale</th><th>Accepted format</th><th>Unsupported browsers</th></tr>
+  <tr><td>Date</td><td>German/English</td><td>DD:MM:YYYY</td><td>n/a</td></tr>
+  <tr><td>Time</td><td>n/a</td><td>hh:mm</td><td>Firefox, Safari</td></tr>
+  <tr><td>Color</td><td>n/a</td><td>#xxxxxx (hex color)</td><td>Safari</td></tr>
+  <tr><td>Item / Thing / Script / Password</td><td>n/a</td><td>Text</td><td>n/a</td></tr>
```

Comment:
Instead of "Text" say "UID of the entity" for Item / Thing. Remove Script and Password, we do not have any special format requirements defined for them, right?